src/Pages/hth/MainHTH/Components/Article/Article.js

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import "./Article.scss";
+import Comment from "./Components/Comment/Comment";
+
+class Article extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      nickname: "kingth_man",
+      inputValue: "",
+      comments: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/hth/data.json", {
+      method: "GET",
+    })
```

Comment:
```suggestion
    fetch("http://localhost:3000/data/hth/data.json")
```
- default가 GET이기 때문에 생략 가능합니다.